src/classes/CON_Preferred_Email_TEST.cls

```diff
@@ -0,0 +1,2215 @@
+            new Hierarchy_Settings__c(Disable_Preferred_Email_Enforcement__c = false)
+        );
+
+        List<Contact> testContactList = UTIL_UnitTestData_TEST.getMultipleTestContacts(2);
+        testContactList[0].WorkEmail__c = '';
+        testContactList[0].Preferred_Email__c = '';
+        testContactList[1].WorkEmail__c = '';
+        testContactList[1].Preferred_Email__c = '';
+        
+        insert testContactList;
+        testContactList[0].Email = 'test@sf.org';
+        testContactList[1].Email = 'test@sf.org';
+        Test.startTest();
+            update testContactList;
+        Test.stopTest();
+
+        List<Contact> updatedContactList = [SELECT Id, AlternateEmail__c, Preferred_Email__c FROM Contact];
+
+        System.assertEquals('test@sf.org', updatedContactList[0].AlternateEmail__c, 'Alternate Email should be same as standard email');
+        System.assertEquals('Alternate Email', updatedContactList[0].Preferred_Email__c, 'Preferred Email should be alternate email');
+        System.assertEquals('test@sf.org', updatedContactList[1].AlternateEmail__c, 'Alternate Email should be same as standard email');
+        System.assertEquals('Alternate Email', updatedContactList[1].Preferred_Email__c, 'Preferred Email should be alternate email');
+        
+    }
+    
+    /*********************************************************************************************************************************
+    * @description Tests Updating a Contact's standard Email field will not update Alternate Email
+    * and Preferred Email when Preferred Email Enforcement is disabled
+    *********************************************************************************************************************************/
+    @isTest 
+    static void standardEmailNotBlankOnUpdateNegativeTest() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(
+            new Hierarchy_Settings__c(Disable_Preferred_Email_Enforcement__c = true)
+        );
+
+        List<Contact> testContactList = UTIL_UnitTestData_TEST.getMultipleTestContacts(2);
+        testContactList[0].WorkEmail__c = '';
+        testContactList[0].Preferred_Email__c = '';
+        testContactList[1].WorkEmail__c = '';
+        testContactList[1].Preferred_Email__c = '';
+        
+        insert testContactList;
+        testContactList[0].Email = 'test@sf.org';
+        testContactList[1].Email = 'test@sf.org';
+
+        Test.startTest();
+            update testContactList;
+        Test.stopTest();
+
+        List<Contact> updatedContactList = [SELECT Id, AlternateEmail__c, Preferred_Email__c FROM Contact];
+
+        System.assertEquals(null, updatedContactList[0].AlternateEmail__c, 'Alternate Email should be cleared');
+        System.assertEquals(null, updatedContactList[0].Preferred_Email__c, 'Preferred Email should be cleared');
+        System.assertEquals(null, updatedContactList[0].AlternateEmail__c, 'Alternate Email should be cleared');
```

Comment:
@skosgi1 should this be updatedContactList[1] instead of 0? Looks like we are repeating the assertion on the same record. 
Also verifying here that we don't need to worry about the order right?

Reply:
Yes, this should be 1, will update that now, thanks for catching that. Order doesn't matter here since values in both the records should be cleared.

Reply:
Updated, resolving.